### PR TITLE
Add Turkish translations for new login form

### PR DESCRIPTION
**Pull request by mustafau**
Here is a screenshot of the login form with Turkish translations.

Screen Shot 2021-04-04 at 23 08 39

### CODE REVIEW

**Review by daniel-beck — APPROVED** [review]
Thanks!

Google Translate approves, and I'm not aware of other contributors who could review this.

**Review by oleg-nenashev — APPROVED** [review]
For larger changes, I can get a review from a native speaker. In this case the localizations look fine

**Comment by oleg-nenashev** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Review by barissimsek — APPROVED** [review]
lgtm (native speaker)

**Comment by mcansozeri** [comment]
first meaning of "Create" is "oluşturmak" in Turkish, second one is "yaratmak". So "oluşturmak" is better transaction.

File: core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_tr.properties [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Welcome\ to\ Jenkins\!=Jenkins''e Ho\u015Fgeldiniz!
```
Comment (urklc) [inline core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_tr.properties:23]: 'Hoş geldiniz' ayrı yazılmalı

File: core/src/main/resources/jenkins/model/Jenkins/login_tr.properties [hunk 2]
```diff
@@ -25,3 +25,7 @@ User=Kullan\u0131c\u0131
 Password=\u015eifre
 Remember\ me\ on\ this\ computer=Beni bu bilgisayarda hat\u0131rla
 login=giri\u015f
+Invalid\ username\ or\ password=Kullan\u0131c\u0131 ad\u0131 veya \u015Fifre yanl\u0131\u015F
```
Comment (urklc) [inline core/src/main/resources/jenkins/model/Jenkins/login_tr.properties:28]: Aslında password'un asıl karşılığı 'parola', çok yaygın bir hata bu.

Comment (mustafau) [reply core/src/main/resources/jenkins/model/Jenkins/login_tr.properties:28]: It's translated as "şifre" in all other places in Jenkins and it's not in scope of this PR to change that. It should be changed in a follow up/dedicated PR.

File: core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_tr.properties (hunk 1 above)
Comment (mustafau) [reply core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_tr.properties]: Fixed. Thanks.

**Review by urklc — APPROVED** [review]
lgtm

**Comment by mustafau** [comment]
ping @oleg-nenashev

**Comment by oleg-nenashev** [comment]
Sorry, we had some issues with the failing test due to the expired Java 11 telemetry timeout. We will merge all ready PRs towards next weekly after https://github.com/jenkinsci/jenkins/pull/5398

**Pull request merged**